docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '2'
+services:
+
+  nginx:
+    restart: always
+    build:
+      context: ./docker/nginx_custom/
+      dockerfile: Dockerfile
+    links:
+      - web:web
+    ports:
+      - "80:80"
+
+  web:
+    build: .
```

Comment:
I'll actually split up Compose files into production and development/testing. I don't think restarts are needed in former.